### DISCUSSION TITLE
Any reason why DatePicker's API doesn't include onBlur prop?

### DISCUSSION THREAD

**Discussion by Powell-v2** [mantinedev/mantine#1316] [Feature requests]
Really curious about this. I guess there's a good reason not to tap into this type of event?

**Comment by rtivital** [comment]
DatePicker has onBlur prop – https://codesandbox.io/s/elegant-water-8o0zx5?file=/src/App.tsx

**Comment by Powell-v2** [comment]
True that! Something must've been up yesterday with the fast refresh in my local dev environment ... Thanks for confirming!
